MediaBrowser.Controller/Entities/BaseItem.cs

```diff
@@ -2085,14 +2085,17 @@ public void AddStudio(string name)
 
             if (!current.Contains(name, StringComparer.OrdinalIgnoreCase))
             {
-                if (current.Length == 0)
+                int curLen = current.Length;
+                if (curLen == 0)
                 {
                     Studios = new[] { name };
                 }
                 else
                 {
-                    var list =
-                    Studios = current.Concat(new[] { name }).ToArray();
+                    var newArr = new string[curLen + 1];
+                    current.CopyTo(newArr, 0);
```

Comment:
can you explain what was wrong with `current.Concat`?

Reply:
It’s way slower and causes more allocations.